ddtrace/contrib/pylibmc/client.py

```diff
@@ -110,7 +115,17 @@ def _trace_cmd(self, method_name, *args, **kwargs):
             if span and args:
                 span.set_tag_str(memcached.QUERY, "%s %s" % (method_name, args[0]))
 
-            return method(*args, **kwargs)
+            if method_name == "get":
+                result = method(*args, **kwargs)
+                span.set_metric(db.ROWCOUNT, 1 if result else 0)
+                return result
+            elif method_name == "gets":
+                result = method(*args, **kwargs)
+                # returns a tuple object that may be (None, None)
+                span.set_metric(db.ROWCOUNT, 1 if result[0] else 0)
```

Comment:
Is result guaranteed to have at least one element? If not this will throw an index error. Also should we count all the elements in the array?

Reply:
Gets is only a single "row" command, and returns a tuple mapping of its value and an ID. See [API](https://sendapatch.se/projects/pylibmc/reference.html#pylibmc.Client.gets:~:text=%2C%20cas_id)-,%C2%B6,-Get%20key%20and) here. Should be guaranteed but I guess we could check for length first to prevent a possible index error to be safe.